src/domains/gps-common/03_gpd_storage_account.tf

```
@@ -0,0 +1,84 @@
+module "gpd_sa" {
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//storage_account?ref=v7.18.0"
+
+  name                = replace("${local.project}-gpd-sa", "-", "")
```

please change the storage name to 

```suggestion
  name                = replace("${local.project}-gpd-sa-sftp", "-", "")
```